main/lz4/APKBUILD

```diff
@@ -1,13 +1,14 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=lz4
-pkgver=1.8.1
+pkgver=1.8.1.2
 pkgrel=0
 pkgdesc="LZ4 is lossless compression algorithm with fast decoder @ multiple GB/s per core."
 url="https://github.com/lz4/lz4"
 arch="all"
 license="BSD"
 checkdepends="diffutils"
+options="!check" # tests take too long for Travis
```

Comment:
Hm, that doesn't necessarily mean that we can't run them on the builders. Is it possible to run a subset of the test suite?

Reply:
rebased with a shorter test.

I run the full test whenever I update the package